src/renderer/components/settings/TitleBarPicker.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import { Margins } from "@vencord/types/utils";
+import { Forms, Select } from "@vencord/types/webpack/common";
+import { isMac, isWindows } from "renderer/utils";
+
+import { SettingsComponent } from "./Settings";
+
+export const TitleBarPicker: SettingsComponent = ({ settings }) => {
+    return (
+        <>
+            <Forms.FormText className={Margins.bottom8}>
+                Customize apps title bar. Pick Discord if you want to use Discord's custom title bar. Requires a full
+                restart
+            </Forms.FormText>
+
+            <Select
+                placeholder="Hidden"
+                options={[
+                    ...(isMac ? [{ label: "Hidden", value: "hidden", default: isMac }] : []),
```

Comment:
i think this is what this setting means?
```suggestion
                    ...(isMac ? [{ label: "Hidden with inset traffic light buttons", value: "hidden", default: isMac }] : []),
```